surveys/sphinx-survey-2025.yml

```diff
@@ -174,10 +174,10 @@ questions:
   multiple: true
   other: true
   choices:
-  - "Google format (indented sections)"
+  - "Google format (my_param (type): description)"
+  - "Numpydoc format (my_param : type \n description indented on new line)"
+  - "Sphinx format (:param [my_param]: description)"
```

Comment:
The parameter formatting may be more easy to recognize. Alternatively can we have links (e.g. to https://www.sphinx-doc.org/en/master/tutorial/automatic-doc-generation.html) or maybe a small sample page that we generate.